tests/baselines/reference/moduleAugmentationImportsAndExports3.errors.txt

```diff
@@ -28,8 +27,6 @@ tests/cases/compiler/f3.ts(13,16): error TS4000: Import declaration 'C' is using
         import {B} from "./f2";
         ~~~~~~
 !!! error TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.
-                        ~~~~~~
-!!! error TS2307: Cannot find module './f2'.

```

Comment:
These follow-on augmentation errors are now gone (or they can be). Previously, since the imports weren't in a source file, they'd never get processed and never get their names resolved (so the files they reference never get included). Now they still aren't resolved up-front, but inside the checker we can now resolve the name late, and so it does so (relatively to the containing file). I find this to be a better output for this scenario, but it shouldn't particularly matter since it is an error anyway.